File: DotNet/CesiumLanguageWriter/Advanced/ICesiumVerticalOriginValuePropertyWriter.cs
namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a vertical origin.
    /// </summary>
    public interface ICesiumVerticalOriginValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value as a vertical origin.
        /// </summary>
        /// <param name="value">The vertical origin.</param>
        void WriteVerticalOrigin(CesiumVerticalOrigin value);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumVerticalOriginValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;
import javax.annotation.Nonnull;

/**
 * A writer that can write a value as a vertical origin.
 */
@SuppressWarnings({
    "unused",
    "deprecation",
    "serial"
})
public interface ICesiumVerticalOriginValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    * Writes the value as a vertical origin.
    * @param value The vertical origin.
    */
    void writeVerticalOrigin(@Nonnull CesiumVerticalOrigin value);
}